src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const getSum = (num) => {
+    if (num === undefined) {
```

Comment:
will it work like this?
```suggestion
  const getSum = (num = undefined) => {
    if (!num) {
```